client/landing/stepper/utils/flow-redirect-handler.ts

```diff
@@ -0,0 +1,50 @@
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+
+// Flows to redirect
+const redirectRoutes = [
+	{ flow: '/setup/blog/', to: '/start:lang?' },
+	{ flow: '/setup/free/', to: '/start/free:lang?' },
+	{ flow: '/setup/link-in-bio/', to: '/start:lang?' },
+	{ flow: '/setup/videopress/', to: '/start:lang?' },
+	{ flow: '/setup/sensei/', to: ':lang?/plugins/sensei-pro/' },
```

Comment:
Minor (non-blocking): Both properties are now actual URLs, so maybe we can rename `flow` to `from`.

Reply:
Updated